Back-End/api/__init__.py

```diff
@@ -0,0 +1,7 @@
+from flask import Flask
+
+def create_app():
+    app = Flask(__name__)
+    app.config['SECRET_KEY'] = 'Genshin Impact is ass'
```

Comment:
I don't mean to be a killjoy but this is something we're handing in for class and that's publicly available so could you please refrain from having curse words in the actual commits? It doesn't matter if it's on your local machine but please leave it out of GitHub